src/smexperiments/experiment.py

```diff
@@ -259,10 +259,11 @@ def delete_all(self, action):
                             trial_component_name=trial_component_summary.trial_component_name,
                         )
                         tc.delete(force_disassociate=True)
-                        t.remove_trial_component(tc)
                         # to prevent throttling
-                        time.sleep(0.2)
+                        time.sleep(1.2)
                     t.delete()
+                    # to prevent throttling
+                    time.sleep(1.2)
```

Comment:
throttling on what? the earlier sleep looks like its for throttling on DisassociateTrialComponent

Reply:
Throttle on .delete() call below (and the .load call in the loop as well).

Reply:
DeleteTrial throttling time is also 1s. This is to prevent trial deletion throttling.